### ISSUE TITLE
cant seem to mine.

### ISSUE THREAD

**Issue by beyond420** [jimathy/jim-mining#33]
i have no errors server side or client side. i can go buy the laser drill and pickaxe but when i go inside to third eye the stones it doesnt turn blue or allow me to do anything. could you possibly help me with this issue please? thank you in advance

**Comment by AppleSatin** [comment]
Having same issue

**Comment by jimathy** [comment]
The most common resolution for this issue is updating or reinstalling qb-target

**Comment by beyond420** [comment]
So im using the newest qb-target but an older core version because when I update the core to the newest which everything will work my burgershot script I can't find a work sround to make it work so I've had to use an older core version and the burgershot will work but my mining and panning will not any suggestions?

**Comment by beyond420** [comment]
ive got it working but for some reason wont appear for everyone to use with third eye only me as a god and if i buy a pickaxe or something it wont show until i left and came back then it would allow me to third eye and use it. any suggestions?

**Comment by RisingRoleplay** [comment]
I have an older version that works fine, however if I try using this version (even with the most recent qb-target still seems to not work, with no errors on either side.

**Comment by midz8** [comment]
Yeah I seem to have this problem with thelatest qb-target. I suggest using the older version or hopefully the script owner finds a work around for this would be great.

**Comment by tmwtvtwitch** [comment]
having the same issue myself, tried using my older version of QB-target and updated to the newest version both to no avail.

**Comment by tmwtvtwitch** [comment]
using this version of qb-target fixed it for me.


https://drive.google.com/file/d/1aB6Nbq2kSpj5-5oGTmFKaDYig5UGH7Xq/view?usp=share_link

**Comment by FrequencyRoleplay** [comment]
@tmwtvtwitch "We're not worthy"

**Comment by tmwtvtwitch** [comment]
> @tmwtvtwitch "We're not worthy"

LOL don't stress it, just figured someone would have given a link, so i took it upon myself to help the next guy having the same issue i was having.

**Comment by ShaggieRizla** [comment]
i have the same problem and this is this actually my only error on the server
![Screenshot_7](https://github.com/user-attachments/assets/65019468-da1c-4677-8c77-b90dc713dae4)
